src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+
+Let's say, in this simplified example, that when a user loads the app in their browser, the following happens in each service:
+
+- _Browser_
+  - 1 request each for HTML, CSS, and JavaScript
+  - 1 rendering task, which sets off 2 requests for JSON data
+^
+- _Backend_
+  - 3 requests to serve static files (the HTML, CSS, and JS)
+  - 2 requests for JSON data
+    - 1 requiring a call to the database 
+    - 1 requiring a call to an external API and work to process the results before returning them to the front end
+^
+- _Database Server_
+  - 1 request which requires 2 queries
+    - 1 query to check authentication 
+    - 1 query to get data
+  
+_Note:_ The external API is not listed precisely because it's external, and you therefore can't see inside of it.
+
+In this example, the entire page-loading process, including all of the above, is represented by a single **trace**. That trace would consist of the following **transactions**:
+
+- 1 browser transaction (for page load)
+- 5 backend transactions (one for each request)
+- 1 database server transaction (for the single DB request)
+  
+Each transaction would be broken down into **spans** as follows:
+
+- _Browser Page-load Transaction_: 7 spans
+  - 1 root span representing the entire page load
+  - 1 span each (3 total) for the HTML, CSS, and JS requests
+  - 1 span for the rendering task, which itself contains
+    - 2 child spans, one for each JSON request
+
+Let's pause here to make an important point: Some, though not all, of the browser transaction spans listed have a direct correspondence to backend transactions listed earlier. Specifically, each request _span_ in the browser transaction corresponds to a separate request _transaction_ in the backend. In this situation, when a span in one service gives rise to a transaction in a subsequent service, we call the original span a parent span to _both_ the transaction and its root span.
+
+In our example, every transaction other than the initial browser page-load transaction is the child of a span in another service, which means that every root span other than the browser transaction root has a parent span (albeit in a different service). In a fully-instrumented system (one in which every service has tracing enabled) this pattern will always hold true: _the only parentless span will be the root of the initial transaction, and of the remaining parented spans, every one will live in the same service as its parent, except for the root spans, whose parents will live in a previous service_. This is worth noting because it is the one way in which transactions are not perfect trees - their roots can (and mostly do) have parents.
```

Comment:
Reworded the sentence to break it up. I ❤️ complex sentences, but they're not always the best option for clarity.

As to paragraphs being one line/many lines: I think I would feel better about that if we had tooling to rewrap automatically (which we don't, to my knowledge). IMHO GitHub should just fix their diff rendering. 😁  (Because I agree, it's a pain that you have to separately call out the change.)